modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -127,11 +135,13 @@ export default class Tile3DLayer extends CompositeLayer {
   }
 
   _updateTileset(tileset3d) {
-    const {timeline, viewport} = this.context;
-    if (!timeline || !viewport || !tileset3d) {
+    const {timeline} = this.context;
+    const {activatedViewports = {}} = this.state;
+    const viewportsNumber = Object.keys(activatedViewports).length;
+    if (!timeline || !viewportsNumber || !tileset3d) {
```

Comment:
Why do you need to check `timeline` here?

Reply:
It is not my peace of code